test/system/070-build.bats

```diff
@@ -30,6 +30,11 @@ EOF
 }
 
 @test "podman buildx - basic test" {
+    driver=$(run_podman info --format "{{.Store.GraphDriverName}}")
```

Comment:
This does not work. One cannot use `run_podman` this way. You need to do:
```bash
    run_podman info blah blah
    if [[ "$output" == "vfs" ]]; then ...skip
```

Reply:
ahh ok, I'll fix it.